easybuild/easyconfigs/g/GRASS/GRASS-7.4.0-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,99 @@
+easyblock = 'ConfigureMake'
+
+name = 'GRASS'
+version = '7.4.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "http://grass.osgeo.org"
+description = """The Geographic Resources Analysis Support System - used
+ for geospatial data management and analysis, image processing,
+ graphics and maps production, spatial modeling, and visualization"""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['http://grass.osgeo.org/grass%s/source' % ''.join(version.split('.')[:2])]
+sources = [SOURCELOWER_TAR_GZ]
+patches = [
+    '%(name)s-%(version)s_blas_lapack.patch',
+]
+checksums = [
+    'cb6fa188e030a3a447fc5451fbe0ecbeb4069ee2fd1bf52ed8e40e9b89e293cc',  # grass-7.4.0.tar.gz
+    '4a3e9a5cf52ac299ea98aac209755ce48b92aea6a9bcd4d838647cd85dd163a4',  # GRASS-7.4.0_blas_lapack.patch
+]
+
+builddependencies = [
+    ('flex', '2.6.4'),
+    ('Bison', '3.0.4'),
+    ('Autotools', '20170619'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('bzip2', '1.0.6'),
+    ('gettext', '0.19.8.1'),
+    ('Python', '2.7.14'),
+    ('libxml2', '2.9.4'),
+    ('libpng', '1.6.32'),
+    ('libreadline', '7.0'),
+    ('ncurses', '6.0'),
+    ('netCDF', '4.5.0'),
+    ('GDAL', '2.2.2', versionsuffix),
+    ('libspatialite', '4.3.0a'),
+    ('GEOS', '3.6.2', versionsuffix),
+    ('PROJ', '4.9.3'),
+    ('SQLite', '3.20.1'),
+    ('freetype', '2.8'),
+    ('FFmpeg', '3.4.1'),
+    ('LibTIFF', '4.0.9'),
```

Comment:
@hajgato There's a conflict on the `LibTIFF` version here, which may be a PITA to resolve now...
```
eb --check-conflicts --from-pr 5840
== temporary log file in case of crash /Users/kehoste/work/TMP/eb-x0sTI7/easybuild-pAWM1z.log

WARNING: Deprecated functionality, will no longer work in v5.0: Use of 'dummy' toolchain is deprecated, use 'system' toolchain instead; see http://easybuild.readthedocs.org/en/latest/Deprecated-functionality.html for more information


WARNING: Deprecated functionality, will no longer work in v5.0: Use of 'dummy' toolchain is deprecated, use 'system' toolchain instead; see http://easybuild.readthedocs.org/en/latest/Deprecated-functionality.html for more information

Conflict between (dependencies of) easyconfigs: LibTIFF-4.0.8-intel-2017b vs LibTIFF-4.0.9-intel-2017b
	LibTIFF-4.0.8-intel-2017b as dep of: GDAL-2.2.2-intel-2017b-Python-2.7.14, GRASS-7.4.0-intel-2017b-Python-2.7.14
	LibTIFF-4.0.9-intel-2017b as dep of: GConf-3.2.6-intel-2017b, GRASS-7.4.0-intel-2017b-Python-2.7.14, GTK+-2.24.32-intel-2017b, Gdk-Pixbuf-2.36.11-intel-2017b, wxPython-3.0.2.0-intel-2017b-Python-2.7.14
Conflict found for dependencies of GRASS-7.4.0-intel-2017b-Python-2.7.14: LibTIFF-4.0.8-intel-2017b vs LibTIFF-4.0.9-intel-2017b
	LibTIFF-4.0.8-intel-2017b as dep of: GDAL-2.2.2-intel-2017b-Python-2.7.14, GRASS-7.4.0-intel-2017b-Python-2.7.14
	LibTIFF-4.0.9-intel-2017b as dep of: GConf-3.2.6-intel-2017b, GRASS-7.4.0-intel-2017b-Python-2.7.14, GTK+-2.24.32-intel-2017b, Gdk-Pixbuf-2.36.11-intel-2017b, wxPython-3.0.2.0-intel-2017b-Python-2.7.14
ERROR: One or more conflicts detected!
```

I would vote for closing this, since it's mostly superseded by #7489 (which is `foss` though, not `intel`)...

Reply:
@boegel I do not mind if we close it. The blas_lapack.patch was changed to a sed in PR #7489, and the Intel version does not need the pthread patch, so it should be pretty forward to make an Intel version from the foss. (sure, sure...) Honestly, I do not have the intention the fix the mess with the `LibTIFF` issue in an old toolchain which was not originally designed to avoid duplicates with different versions...